<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manual_scales</class>
 <widget class="QDialog" name="manual_scales">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="hzl_ampl">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Amplitude min/max</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="a_min"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="a_max"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hzl_phz">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Phase min/max         </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="phz_min"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="phz_max"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hzl_freq">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Freq min/max            </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="f_min"/>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="f_max"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>manual_scales</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>manual_scales</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
